recipes/pandera/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "pandera" %}
+{% set version = "0.2.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 288e4be03e5072612bbc58623e224d5b915b80913f8fa359cb01c1e9b95afa86
+
+build:
+  number: 0
+  skip: True  # [py<35]
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - numpy
+    - pandas
+    - scipy
+    - wrapt
+
+test:
+  imports:
+    - pandera
+
+about:
+  home: https://github.com/pandera-dev/pandera
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
```

Comment:
Also, since you are the developer of the package, it would be nice if you can add the License file in the pypi distribution, so we don't need to add it here.

Reply:
Please add the line `license_file: LICENSE.txt` back. What I meant was that the file `LICENSE.txt` from the PR can be removed since that file is in the distribution now (sorry for not being clear about that).

Reply:
gotcha! thanks for clarifying. added it back